float64[] positions
int64 time
string[] joint_names
int64 status_hand
